main.go

```diff
@@ -216,3 +230,39 @@ func main() {
 		fmt.Println(string(d))
 	}
 }
+
+func initConfigDir(path string) error {
+	//Create directory if not exist
+	if _, err := os.Stat(path); errors.Is(err, fs.ErrNotExist) {
+		if errMkdir := os.MkdirAll(path, os.ModeDir); errMkdir != nil {
+			return errMkdir
+		}
+	} else if err != nil {
+		return err
+	}
+
+	//Create all files if doesn't exist
+	files := []string{ //Add some config file here
+		logFile,
+	}
```

Comment:
log file creation is handled by the `newLogger` function. let's keep it at one place